### ISSUE TITLE
Accessibility guidelines (first draft) 

### ISSUE THREAD

**Issue by LHinson** [patternfly/patternfly-next#107]
Introduce a draft of the accessibility guidelines to this repo. 

- [x] Schedule a meeting with those that are interested in being involved 
- [x] Need to determine where these guidelines would be introduced (contributing.md?)
- [x] Provide an outline of what we want to provide and how we will provide that.

**Comment by jgiardino** [comment]
> Need to determine where these guidelines would be introduced (contributing.md?)

For now, we'll include an .md file for the Accessibility Guide, with a link to it from the README.md. 

> Provide an outline of what we want to provide and how we will provide that.

For now, the [first draft is in a google doc](https://docs.google.com/document/d/1tlhfKHNagj-t9vK1juI3i-IcEXmhpYQfKr2yZhPH_gY/edit?usp=sharing) and has been posted in the patternfly-a11y slack channel for feedback. When I return from PTO, I'll incorporate any feedback I have and contribute the .md file.

**Comment by jgiardino** [comment]
Yes, I agree. @andresgalante, @matthewcarleton and I had talked about ultimately having an A11y guide that's available within the workspace. The .md file is a temporary solution until we have the workspace set up to support that.

This is what we had with the old workspace: https://pf-next.com/guidelines/. I've lost track of where we're at with the new workspace, and whether that's already available?
